fiasko_bro/code_validator.py

```diff
@@ -136,6 +137,8 @@ class CodeValidator:
         'max_pep8_line_length': 100,
         'max_number_of_lines': 200,
         'max_indentation_level': 4,
+        'max_num_of_py_files': 100,
+        'check_repo_size': True,
```

Comment:
I know it's confusing, but the `_default_settings` dictionary is intended for validator arguments. If you want to pass an argument to `validate`, just do it with a standard optional argument.